src/components/10-atoms/carousel/index.js

```diff
@@ -33,8 +35,11 @@ class AXACarousel extends InlineStyles {
       autorotatedisabled: { type: Boolean },
       autorotatetime: { type: Number, defaultValue: 5000 },
       keysenabled: { type: Boolean },
+      visible: { type: Number, reflect: true },
+      arrowstyle: { type: String, reflect: true },
+      mode: { type: String, reflect: true },
       _animationWrapperClass: { type: String },
-      _carouselMinHeight: { type: Number },
+      _carouselMinHeight: { type: Number, defaultValue: 0 },
```

Comment:
Why an "internal prop" and not just a class property in the constructor?

Reply:
As you can see in the diff, this was not introduced by me, so ask my predecessor ;-)

Reply:
Only props are watched by lit-element and are updated automatically the template. I think that was the reason.